File: domain/commands/registry.py
from domain.commands.handlers import (
    CommandHandler,
    handle_config,
    handle_exit,
    handle_help,
    handle_toggle_logging,
)

COMMAND_REGISTRY: dict[str, CommandHandler] = {
    "/exit": handle_exit,
    "/config": handle_config,
    "/help": handle_help,
    "/toggle_logging": handle_toggle_logging, 
}

File: src/adapters/cli/commands/registry.ts
import type { CommandHandler } from "./handlers.js";
import {
  handleChangeWorkingDirectory,
  handleConfig,
  handleExit,
  handleHelp,
  handleModel,
  handleToggleLogging,
} from "./handlers.js";

export const COMMAND_REGISTRY: Record<string, CommandHandler> = {
  "/exit": handleExit,
  "/config": handleConfig,
  "/help": handleHelp,
  "/model": handleModel,
  "/toggle_logging": handleToggleLogging,
  "/cd": handleChangeWorkingDirectory,
};
